contrib/docker-machine-install-bundle.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+#
+# This script installs the bundle to Docker Machine instances, for the purpose
+# of testing the latest Docker with Swarm mode enabled.
+# Do not use in production.
+#
+# Requirements (on host to run this script)
+#  - bash is installed
+#  - Docker Machine is installed
+#  - GNU tar is installed
+#
+# Requirements (on Docker machine instances)
+#  - Docker can be managed via either `systemctl` or `service`
+#
+set -e
+
+errexit() {
+    echo "$1"
+    exit 1
+}
+
+BUNDLE="bundles/$(cat VERSION)"
+
+bundle_files(){
+    # prefer dynbinary if exists
+    for f in dockerd docker-proxy; do
+	if [ -d $BUNDLE/dynbinary-daemon ]; then
+	    echo $BUNDLE/dynbinary-daemon/$f
+	else
+	    echo $BUNDLE/binary-daemon/$f
+	fi
+    done
+    for f in docker-containerd docker-containerd-ctr docker-containerd-shim docker-init docker-runc; do
+	echo $BUNDLE/binary-daemon/$f
+    done
+    if [ -d $BUNDLE/dynbinary-client ]; then
+	echo $BUNDLE/dynbinary-client/docker
+    else
+	echo $BUNDLE/binary-client/docker
+    fi
+}
+
+control_docker(){
+    m=$1; op=$2
+    # NOTE: `docker-machine ssh $m sh -c "foo bar"` does not work
+    #       (but `docker-machine ssh $m sh -c "foo\ bar"` works)
+    #       Anyway we avoid using `sh -c` here for avoiding confusion
+    script="if hash systemctl 2> /dev/null; then systemctl $op docker; else service docker $op; fi"
```

Comment:
Indeed, although there is a symlink included at `/etc/init.d/docker` for better compatibility/discoverability: https://github.com/boot2docker/boot2docker/blob/bc7888a03a6cb0480bfdf31a7b615e516d0afe52/rootfs/make_iso.sh#L17 :smile: :+1: